src/vis.ts

```diff
@@ -1,38 +1,63 @@
 import * as THREE from 'three';
-import * as plane from './plane';
+import {Plane} from './plane';
 import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
 
 export class Vis {
   renderer: THREE.WebGLRenderer;
   scene: THREE.Scene;
   camera: THREE.PerspectiveCamera;
-  aircraft: THREE.Group;
+  aircraftModels: THREE.Group;
   camControls: OrbitControls;
+  static readonly camFov = 70;
+  static readonly camAspectDefault = 1; //should be overwritten on attach or resize
+  static readonly camNearFrustum = 0.01;
+  static readonly camFarFrustum = 10;
 
   constructor() {
-    const width = window.innerWidth,
-      height = window.innerHeight;
-
-    this.camera = new THREE.PerspectiveCamera(70, width / height, 0.01, 10);
+    this.camera = new THREE.PerspectiveCamera(
+      Vis.camFov,
+      Vis.camAspectDefault,
+      Vis.camNearFrustum,
+      Vis.camFarFrustum
+    );
     this.camera.position.z = 1;
 
     this.scene = new THREE.Scene();
 
-    this.aircraft = new THREE.Group();
-    this.aircraft.name = 'aircraft';
-    this.scene.add(this.aircraft);
+    this.aircraftModels = new THREE.Group();
+    this.aircraftModels.name = 'aircraftModels';
+    this.scene.add(this.aircraftModels);
 
     this.renderer = new THREE.WebGLRenderer({antialias: true});
     this.camControls = new OrbitControls(this.camera, this.renderer.domElement);
     this.camControls.update();
-    this.renderer.setSize(width, height);
-    this.renderer.setAnimationLoop(this.animation);
+
+    this.renderer.setAnimationLoop(time => {
+      this.drawFrame(time);
+    });
   }
-  animation = (time: DOMHighResTimeStamp) => {
+
+  drawFrame(time: DOMHighResTimeStamp) {
     // time is dead for now but will be used for physics sim
     this.renderer.render(this.scene, this.camera);
-  };
-  addAircraft(planein: plane.Plane) {
-    this.aircraft.add(planein.model);
+  }
+
+  addaircraftModels(plane: Plane) {
+    this.aircraftModels.add(plane.model);
+  }
+
+  attachTo(element: HTMLElement) {
+    element.appendChild(this.renderer.domElement);
+    this.resize(element);
+    window.onresize = () => {
+      this.resize(element);
+    };
+  }
+  resize(element: HTMLElement) {
```

Comment:
Try to be consistent w/ empty lines between methods, there's not one here but there is one above. I believe google style guide prefers an empty line between every method.